las/client.py

```diff
@@ -2431,3 +2450,35 @@ def delete_workflow_execution(self, workflow_id: str, execution_id: str) -> Dict
  :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
         """
         return self._make_request(requests.delete, f'/workflows/{workflow_id}/executions/{execution_id}')
+
+    def list_roles(self, *, max_results: Optional[int] = None, next_token: Optional[str] = None) -> Dict:
+        """List roles available, calls the GET /roles endpoint.
+
+        :param max_results: Maximum number of results to be returned
+        :type max_results: int, optional
+        :param next_token: A unique token for each page, use the returned token to retrieve the next page.
+        :type next_token: str, optional
+        :return: Roles response from REST API without the content of each role
+        :rtype: dict
+
+        :raises: :py:class:`~las.InvalidCredentialsException`, :py:class:`~las.TooManyRequestsException`,\
+ :py:class:`~las.LimitExceededException`, :py:class:`requests.exception.RequestException`
+        """
+        params = {
+            'maxResults': max_results,
+            'nextToken': next_token,
```

Comment:
Do we check in GET /roles if `maxResults` and `nextToken` is None, so that it is safe to send it here if they are None?

Reply:
They are stripped later by the requests library